app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/RewardCarouselScreen.kt

```diff
@@ -144,19 +144,24 @@ fun RewardCarouselScreen(
             ) { reward ->
 
                 val ctaButtonEnabled = when {
+                    RewardUtils.isNoReward(reward) && (project.postCampaignPledgingEnabled() ?: false && project.isInPostCampaignPledgingPhase() ?: false) -> true
```

Comment:
fyi, no need to act on it now, but all this logic could be moved to an extension function or the rewards utils, and unit tested, so we are sure future changes won't break anything